integration_tests/mockApis/outOfServiceBed.ts

```diff
@@ -154,29 +181,48 @@ export default {
       })
     ).body.requests,
   verifyOutOfServiceBedsDashboard: async ({
-    page = '1',
-    sortBy = 'outOfServiceFrom',
-    sortDirection = 'asc',
+    page = 1,
+    sortBy,
+    sortDirection,
+    temporality = 'current',
+    premisesId,
+    perPage = 10,
   }: {
-    page: string
+    page: number
```

Comment:
We seem to use strings and numbers in `integration_tests/tests/v2Manage/outOfServiceBeds.cy.ts` so maybe we should have this type as both? ...not that it's typechecking anyway